iac/terraform/aws/auth_cognito/lambdas.tf

```diff
@@ -0,0 +1,38 @@
+locals {
+  lambda_archive_path = "${path.module}/../../../../core/services/aws/.build/zips"
+  lambda_options = {
+    custom_email_template = {
+      name          = "custom-email-template"
+      handler       = "customEmailTemplate.default"
+      zip_path      = "${local.lambda_archive_path}/customEmailTemplate.zip"
+      statement     = local.policies.common_policies
+      env_variables = {}
+    },
+    post_confirmation = {
+      name      = "post-confirmation"
+      handler   = "postConfirmation.default"
+      zip_path  = "${local.lambda_archive_path}/postConfirmation.zip"
+      statement = concat(local.policies.common_policies, local.policies.dynamodb_policy)
+      env_variables = {
+        DYNAMODB_TABLE = "${var.table_name}"
```

Comment:
Should be DYNAMODB_TABLE_NAME instead of DYNAMODB_TABLE

Reply:
Updated variable name. You can find it with the commit message. [commit](https://github.com/CraftsmenLtd/BloodConnect/commit/28f045306270fdc86a03546bd3b3e08356c4dc74#diff-9fcfda62a75664be8204a906a8560f94308c1ba2b6628a3ee928aacda1a7d7f1)